test/logic/models/sort_test.dart

```diff
@@ -112,19 +172,35 @@ void main() {
           songWith(id: 10, dateModified: 0, dateAdded: 0, title: '', artist: 'AA', album: ''),
           songWith(id: 11, dateModified: 0, dateAdded: 0, title: '', artist: 'a', album: ''),
           songWith(id: 12, dateModified: 0, dateAdded: 0, title: '', artist: 'z', album: ''),
+          songWith(id: 21, dateModified: 0, dateAdded: 0, title: '', artist: 'А (кириллица)', album: ''),
+          songWith(id: 22, dateModified: 0, dateAdded: 0, title: '', artist: 'АА (кириллица)', album: ''),
+          songWith(id: 23, dateModified: 0, dateAdded: 0, title: '', artist: 'а (кириллица)', album: ''),
+          songWith(id: 24, dateModified: 0, dateAdded: 0, title: '', artist: 'я (кириллица)', album: ''),
           songWith(id: 5, dateModified: 0, dateAdded: 0, title: 'A', artist: '', album: ''),
           songWith(id: 7, dateModified: 0, dateAdded: 0, title: 'a', artist: '', album: ''),
           songWith(id: 6, dateModified: 0, dateAdded: 0, title: 'AA', artist: '', album: ''),
           songWith(id: 8, dateModified: 0, dateAdded: 0, title: 'z', artist: '', album: ''),
+          songWith(id: 17, dateModified: 0, dateAdded: 0, title: 'А (кириллица)', artist: '', album: ''),
```

Comment:
At first I thought that it's weird that this entry is is sorted after the `z`, but it looks like it's standard behaviour that all cyrillic letters are sorted after the ascii ones.